select first address when focusout

namkn opened the pull request:
Add watchFocusOut option and automatically select first address upon watchFocusOut being true but only if user enter at least 3 characters; otherwise clear the field

namkn committed:
select first address when focusout

Add watchFocusOut option and automatically select first address upon watchFocusOut being true but only if user enter at least 3 characters; otherwise clear the field

vik-singh commented on src/ngAutocomplete.js:
```diff
@@ -43,6 +43,7 @@ angular.module( "ngAutocomplete", [])
                 //options for autocomplete
                 var opts
                 var watchEnter = false
+                let watchFocusOut = false
```
Do not use ES6 in library code

vik-singh commented on src/ngAutocomplete.js:
```diff
@@ -104,6 +110,15 @@ angular.module( "ngAutocomplete", [])
 
                 var locationInput = angular.element(document.getElementById('locations'))
+                locationInput.bind('focusout', function() {
+                    let input = element.val()
```
Same here - use ES5

vik-singh commented on src/ngAutocomplete.js:
```diff
@@ -181,13 +196,6 @@ angular.module( "ngAutocomplete", [])
                                                     scope.details = detailsResult;
-
-                                                    //on focusout the value reverts, need to set it again.
-                                                    var watchFocusOut = element.on('focusout', function(event) {
```
Are we sure this should be removed - what about other cases like full address etc.?

namkn committed:
replace let with var

I don't see any difference in cases like full address. in fact their focusout handling is pretty unpredictable
